N/NetCDFF/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "NetCDFF"
+version = v"4.5.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/Unidata/netcdf-fortran/archive/refs/tags/v4.5.3.tar.gz", "c6da30c2fe7e4e614c1dff4124e857afbd45355c6798353eccfa60c0702b495a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/netcdf-fortran-4.5.3/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; )
+]
+
```

Comment:
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=9914&view=logs&j=9bfd259a-c2a1-53f6-dea5-4be155f9c940&t=8329a79a-e40d-5680-2144-5971d3335d88&l=1215
```
┌ Warning: /agent/_work/1/s/N/NetCDFF/build/x86_64-linux-gnu/M0dgUNVX/x86_64-linux-gnu-libgfortran3-cxx03/destdir/lib/libnetcdff.so.7.0.0 links to libgfortran!  This causes incompatibilities across major versions of GCC.  To remedy this, you must build a tarball for each major version of GCC.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_gfortran_versions(platforms)
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/EyOxg/src/auditor/compiler_abi.jl:60
```
```suggestion
platforms = expand_gfortran_versions(platforms)
```